docs/en/rest_api.md

```diff
@@ -0,0 +1,74 @@
+# REST APIs
+
+## Data Insertion
+
+reqeust url: http://ip:port/dbs/{db_name}/tables/{table_name}
+
+http method: PUT 
+
+request body: 
+```
+{
+    "value": [
+    	[v1, v2, v3]
+    ]
+}
+```
+
++ Only one data can be inserted at a time.
++ The data should be arranged according to the schema strictly.
+
+### Examples
+
+```
+curl http://127.0.0.1:8080/dbs/db/tables/trans -X PUT -d '{
+"value": [
+    ["bb",24,34,1.5,2.5,1590738994000,"2020-05-05"]
+]}'
+```
+response:
+
+```
+{
+    "code":0,
+    "msg":"ok"
+}
+```
+
+## Real-Time Feature Computation
+
+reqeust url: http://ip:port/dbs/{db_name}/deployments/{deployment_name}
+
+http method: POST
+
+request body: 
+
+```
+{
+    "input": [["row0_value0", "row0_value1", "row0_value2"], ["row1_value0", "row1_value1", "row1_value2"], ...],
+    "need_schema": false
+}
+```
+
++ Multiple rows input are supported which correspond to the array of data.data fields in the returned value.

```

Comment:
Multiple rows of input are supported, whose returned values correspond to the fields in the data.data array.

Reply:
Thank you for your suggestions! Should I close this PR and create a new one?

Reply:
no. just push more commits here :-)

Reply:
I am sorry that I have deleted the repo I forked before. I am afraid I can not push commit to this PR, so I created a new PR #1526. Thank you!